backends/qualcomm/passes/fold_qdq.py

```diff
@@ -56,6 +56,9 @@ def _fold(self, graph_module: torch.fx.GraphModule) -> torch.fx.GraphModule:
             for i in range(1, len(n.args)):
                 if type(n.args[i]) == torch.fx.node.Node:
                     to_be_removed.append(n.args[i])
+                    # could be a commonly shared attribute between q & dq
+                    if n.args[i].target == exir_ops.edge.aten._to_copy.default:
+                        to_be_removed.append(n.args[i].args[0])
```

Comment:
This part is necessary, since the q / dq node of parameters in placeholder format will commonly share the same scale and offset. We need to tackle it with this approach to correctly fill in quantization attribute.